curriculum/challenges/english/10-coding-interview-prep/rosetta-code/execute-a-markov-algorithm.md

```diff
@@ -8,7 +8,9 @@ dashedName: execute-a-markov-algorithm
 
 # --description--
 
-Create an interpreter for a [Markov Algorithm](<https://en.wikipedia.org/wiki/Markov algorithm> "wp: Markov algorithm").
+Markov algorithm is a string rewriting system that uses grammar-like <em>rules</em> to operate on strings of symbols. Markov algorithms have been shown to be Turing-complete, which means that they are suitable as a general model of computation and can represent any mathematical expression from its simple notation.
```

Comment:
This sentence is very similar to the one on wikipedia. In order to use their text, we need attribution. 
https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License
I highly recommend paraphrasing the definitions.